ci/get_groups.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Ensure that the script is being executed from its directory
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
+
+# Check if input JSON and field arguments are provided
+if [ "$#" -ne 2 ]; then
+  echo "Usage: ./get_groups.sh <input_json> <field>"
+  exit 1
+fi
+
+# Assign command-line arguments to variables
+input_json="$1"
+field="$2"
+
+output=$(echo $input_json | jq -L . -c --arg field "$field" 'include "group_by_field"; group_by_field($field)')
+echo $output 
+
```

Comment:
```suggestion
# Assign command-line arguments to variables
export input_json="$1"
export field="$2"

jq -nc 'include "group_by_field"; (env.input_json|fromjson) | group_by_field(env.field)'
```

Small suggestions to make this less verbose if you're interested:

- if you `export` the `input_json` and `field` variables, you can access them with `env.<var>` (see [docs](https://jqlang.github.io/jq/manual/#$ENV,env))
- when we added an `include` statement in our `jq`, we didn't need to add the `-L .` flags. you might be able to remove this

**Note**: I didn't test this suggestion.